packages/credential-governance/src/domain-hierarchy.ts

```diff
@@ -214,35 +215,44 @@ export class DomainHierarchy {
     event: EventFilter;
     contractInterface: utils.Interface;
   }) => {
-    const filter = {
-      fromBlock: 0,
-      toBlock: 'latest',
-      address: event.address,
-      topics: event.topics || [],
-    };
-    // console.log("Inside getAllLogs first map " + (new Date()).toString());
-    const logs = await provider.getLogs(filter);
-    const rawLogs = await pMap(logs, async (log) => {
-      const parsedLog = contractInterface.parseLog(log);
-      return contractInterface.decodeEventLog(parsedLog.name, log.data, log.topics);
-    });
-    
-    // console.log("Inside getAllLogs after first map " + (new Date()).toString());
-    
-    // const rawLogs = logs.map((log) => {
-    //   const parsedLog = contractInterface.parseLog(log);
-    //   /** ethers_v5 Interface.parseLog incorrectly parses log, so have to use lowlevel alternative */
-    //   return contractInterface.decodeEventLog(
-    //     parsedLog.name,
-    //     log.data,
-    //     log.topics
-    //   );
-    // });
-    console.log("Inside getAllLogs second map " + (new Date()).toString());
-    const domains = await pMap(rawLogs, parser);
-    console.log("Inside getAllLogs after second map " + (new Date()).toString());
+    console.group('getDomainsFromLogs');
+
+    let mem = process.memoryUsage().heapUsed;
+    let maxMem = mem;
+    console.log(`max mem ${maxMem / 1e6} mb`);
+
+    mem = process.memoryUsage().heapUsed;
+    if (mem > maxMem) {
+      maxMem = mem;
+      console.log(`mem ${maxMem / 1e6} mb`);
+    }
+
+    const logBatchSize = 25_000;
```

Comment:
```suggestion
    const logBatchSize = 25000;
```
@JGiter Is this intentional or by mistake? To run the script locally I changed it to the suggested value else it wasn't ending for me.

Reply:
It was intentional. Such syntax makes big number more readable in my opinion. But I am ok with your suggestion